conformance/tests/annotations_forward_refs.py

```diff
@@ -38,7 +38,7 @@ class ClassA:
 # The following should all generate errors because they are not legal type
 # expressions, despite being enclosed in quotes.
 def invalid_annotations(
-    p1: "eval(" ".join(map(chr, [105, 110, 116])))",  # E
+    p1: "eval(' '.join(map(chr, [105, 110, 116])))",  # E
```

Comment:
```suggestion
    p1: "eval(''.join(map(chr, [105, 110, 116])))",  # E
```
I think this makes slightly more sense:
```
>>> eval(''.join(map(chr, [105, 110, 116])))
<class 'int'>
```
Though in general this is meant as an example of "don't do this", so the exact weird code doesn't matter too much.

Reply:
Thanks for the explanation.